package.json

```diff
@@ -17,7 +18,7 @@
     "dist"
   ],
   "engines": {
-    "node": "~16"
+    "node": ">=16"
```

Comment:
I'm worried, about why we had the 16th node fixed?

Reply:
Probably to keep local node version the same as lambda runtime, but it causes problems with semantic-release.

Reply:
Even when we use semantic release from gh action?

Reply:
Let's see, I'll revert this change for now